src/Components/PostComment.jsx

```diff
@@ -0,0 +1,33 @@
+import { useState } from "react"
+import { postCommentById, fetchUsers } from "../api"
+import { useParams } from "react-router-dom";
+import { optimisticPostedComment } from "../handle";
+export default function PostComment({ comments, setComments}){
+  const { review_id } = useParams();
+  const [postedComment, setPostedComment] = useState('')
+  const [currentUser, setCurrentUser] = useState('grumpy19')
+  const [postedErr, setPostErr] = useState(false)
+
+  const handleSubmit = (e)=>{
+    e.preventDefault();
```

Comment:
When posting I can click submit multiple times and it posts duplicates - we need to add some logic to disable it whilst it is posting 